athom-smart-plug-v2.yaml

```diff
@@ -84,27 +84,31 @@ sensor:
     update_interval: 60s
 
   - platform: cse7766
-    update_interval: 10s
     current:
       name: "Current"
       filters:
-          - lambda: if (x < 0.060) return 0.0; else return x;   #For the chip will report less than 3w power when no load is connected
+        - throttle_average: 10s
+        - lambda: if (x < 0.060) return 0.0; else return x;   #For the chip will report less than 3w power when no load is connected
 
 
     voltage:
       name: "Voltage"
+      filters:
+        - throttle_average: 10s
     power:
       name: "Power"
       id: power_sensor
       filters:
-          - lambda: if (x < 3.0) return 0.0; else return x;    #For the chip will report less than 3w power when no load is connected
```

Comment:
This line should not be removed.

Reply:
It's not actually removed, you can see it still there at 103. I think it looks like a removal and addition either because the throttle_average filter was added above it, or because I updated the indentation to be consistent.

Reply:
Ah ok sounds good I missed that.